other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=-mkdir /var/run/tox-bootstrapd -p
+ExecStartPre=chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
+WorkingDirectory=/var/lib/tox-bootstrapd
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
+User=tox-bootstrapd
+Group=tox-bootstrapd
+PIDFile=/var/run/tox-bootstrapd.pid
+#CapabilityBoundingSet=CAP_NET_BIND_SERVICE
```

Comment:
@stqism, do you know any way to make capabilities work in systemv scripts? As I understand - it's impossible to set capability to script, without setting it to interpreter, and it will not work, if set in filesystem just for daemon file. May be I'm wrong.